ui/lib/pki/addon/components/page/pki-configuration-details.hbs

```diff
@@ -23,8 +23,20 @@
     </ToolbarActions>
   </Toolbar>
 
+  {{#if (not (eq @cluster 403))}}
+    <h2 class="title is-4 has-bottom-margin-xs has-top-margin-xl has-border-bottom-light has-bottom-padding-s">
+      Cluster Config
```

Comment:
normally we lower case the second word, but all of the other headers in this section are capitalized so I followed that pattern here.

Reply:
I followed the designs when putting these headers! Open to changing them so that it matches normal conventions.

Reply:
That's totally fine :) It's something i've been meaning to bring up with design